tutorials/plot_brainstorm_phantom_elekta.py

```diff
@@ -137,3 +138,19 @@
 
 fig.tight_layout()
 plt.show()
+
+
```

Comment:
Why not `plot_dipole_locations`? 

https://mne-tools.github.io/dev/generated/mne.viz.plot_dipole_locations.html#mne.viz.plot_dipole_locations

Reply:
Ahh okay.

In any case, please make this a separate section with `###########` so it renders nicer in SG

Reply:
Ok